packages/phoenix-event-display/src/event-display.ts

```diff
@@ -359,7 +359,7 @@ export class EventDisplay {
    */
   public async loadGLTFGeometry(
     url: any,
-    name: string,
```

Comment:
Have you tested how the behavior is without the name? The name appears in the Phoenix menu and can potentially make it hard to navigate through the menu if there is no name.

Reply:
Test

Reply:
@kjvbrt is trying to reply... but in short, if he does not make it undefined then the passed name overrides the names in the geometry

Reply:
I think we have to accept this.